fluent-bit-kinesis.go

```diff
@@ -0,0 +1,139 @@
+// Copyright 2019 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License"). You may
+// not use this file except in compliance with the License. A copy of the
+// License is located at
+//
+//  http://aws.amazon.com/apache2.0/
+//
+// or in the "license" file accompanying this file. This file is distributed
+// on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either
+// express or implied. See the License for the specific language governing
+// permissions and limitations under the License.
+
+package main
+
+import (
+    "C"
+    "unsafe"
+
+    "github.com/hossain-rayhan/amazon-kinesis-streams-for-fluent-bit/kinesis"
```

Comment:
.

Reply:
what does this dot (.) mean? newline? It was intentional to separate the local and other source dependencies.